src/planscape/plan/views.py

```diff
@@ -254,3 +254,18 @@ def create_project_area(request: HttpRequest) -> HttpResponse:
         return HttpResponse(str(project_area.pk))
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+
+
+def get_project_areas(request: HttpRequest) -> HttpResponse:
+    try:
+        assert isinstance(request.GET['project_id'], str)
+        project_id = request.GET.get('project_id', 0)
```

Comment:
Types look wonky---should 0 be a string "0" ?  Or should the project_id be converted to an int?

Reply:
Modified to default to string "0", thanks!